models/health.go

```diff
@@ -157,3 +163,10 @@ func (ps ProxyStatus) IsSynced() bool {
 func isComponentStatusSynced(componentStatus string) bool {
 	return componentStatus == "Synced"
 }
+
+func GetHealthConfiguration(annotations map[string]string) (string, error) {
```

Comment:
We reserve the "Get*" verbs for backend calls, for these kind of operations a "Parse*" / "Cast*" fits better because it's just extracting data from one format to another.

Reply:
I nned to remove this method is not used :(